selectors.yml

```diff
@@ -1,17 +1,23 @@
 selectors:
   # Testnets
-  31: 
-    selector: 8953668971247136127 
+  31:
+    selector: 8953668971247136127
     name: "bitcoin-testnet-rootstock"
   97:
     selector: 13264668187771770619
     name: "binance_smart_chain-testnet"
   280:
     selector: 6802309497652714138
     name: "ethereum-testnet-goerli-zksync-1"
-  300: 
-    selector: 6898391096552792247 
+  296:
+    selector: 222782988166878823
+    name: "hedera-testnet"
```

Comment:
Same: is this a true L1? And main/only/unnamed testnet?

Reply:
I believe it is a true L1. Here are some links if you could help me confirm.

And yes the testnet is unnamed. They just call it "Hedera Testnet".

[Hedera Website](https://hedera.com/learning/hedera-hashgraph/what-is-hedera-hashgraph)
[Hedera Docs](https://docs.hedera.com/hedera/#set-up-your-environment)
[Hedera Questionnaire](https://docs.google.com/spreadsheets/d/1lTYmYTY8SdFRsU35LOeZqVdDZipI3j06HbQ1BT0QN9c/edit#gid=625078687)

Reply:
Yes, looks like a true L1. Should be good to go. Thanks